### PR TITLE
Homework 2019.05.03

### PR DESCRIPTION
**Pull request by HunanKhachatryan**
Task: Create a page by using the bootstrap.

### CODE REVIEW

File: Hunan_Khachatryan/bootstrap-page.html [hunk 1]
```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
```
Comment (Anjela-Andreasyan) [inline Hunan_Khachatryan/bootstrap-page.html]: Coding style is not followed - nested blocks are not intended correctly. (html/head, head/script, button etc.)

File: Hunan_Khachatryan/bootstrap-page.html [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+</head>
+<body>
+
+<!--NavBar-->
+
+<nav class="navbar navbar-expand-sm navbar-light bg-light ">
+    
+    <a class="navbar-brand text-dark" href="#">Bootstrup 101</a>
+    <button type="button" class="navbar-toggler navbar-toggler-right "  data-toggle="collapse" data-target="#collapsibleNavbar">
```
Comment (Anjela-Andreasyan) [inline Hunan_Khachatryan/bootstrap-page.html]: extra spaces

Comment (HunanKhachatryan) [reply Hunan_Khachatryan/bootstrap-page.html]: The defects have been corrected.

File: Hunan_Khachatryan/bootstrap-page.html (hunk 1 above)
Comment (HunanKhachatryan) [reply Hunan_Khachatryan/bootstrap-page.html]: the defects have been corrected

**Comment by Anjela-Andreasyan** [comment]
screencapture-file-Users-anjela-itc10-homeworks-ITC-10-Hunan-Khachatryan-bootstrap-page-html-2019-05-04-22_08_50
the styles are not matching the to provided design
- sign up button
- footer
- section after Services
- pictures are returning 403 forbidden

**Pull request merged**